### PR TITLE
fix(ENG-3275): Fix failing package examples

### PR DESCRIPTION
**Pull request by caitriona-cloudsmith**
- Updated deprecated CircleCI images 
- Fixed alpine-cli image to run on any underlying architecture

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ orbs:
               name: Prepare build environment
               command: |
                 echo "export BUILD_NUMBER=`/home/circleci/repo/bin/generate-build-number`" >> $BASH_ENV
-                echo "export PATH=/home/circleci/.local/bin:$PATH" >> $BASH_ENV
+                echo 'export PATH=/home/circleci/.local/bin:$PATH' >> $BASH_ENV
```
Comment (dalbertcloudsmith) [inline .circleci/config.yml:34]: Did you intend to to send the variable to $BASH_ENV as opposed to expanding the existing path?

File: .circleci/config.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ orbs:
               name: Prepare build environment
               command: |
                 echo "export BUILD_NUMBER=`/home/circleci/repo/bin/generate-build-number`" >> $BASH_ENV
-                echo "export PATH=/home/circleci/.local/bin:$PATH" >> $BASH_ENV
```
Comment (estebangarcia) [inline .circleci/config.yml:34]: Question: Changing the quotes here will cause `$PATH` to not be expanded, is that intended?

File: README.md [hunk 3]
```diff
@@ -26,13 +26,15 @@ The examples are configured using environment variables. You can set these manua
 ```bash
 $ export CLOUDSMITH_API_KEY=YOUR_API_KEY_HERE
 $ export CLOUDSMITH_REPOSITORY=your-name-or-org/your-repository
+$ export CLOUDSMITH_REPOSITORY_IS_PRIVATE=yes/no
```
Comment (dalbertcloudsmith) [inline README.md:29]: Are the actual correct values "yes" or "no", or is it something different like YES/NO or true/false?

I don't currently know the answer to that.

File: examples/alpine-cli/build.sh [hunk 4]
```diff
@@ -25,4 +24,4 @@ build_script | _docker run -i --name build-$BUILD_NUMBER alpine-example-build sh
 
 # since we build inside a container we need to explicitly copy the artifacts out
 # after building
-_docker cp build-$BUILD_NUMBER:/home/circleci/packages/build/x86_64/ .
+_docker cp -a build-$BUILD_NUMBER:/home/circleci/packages/build/. .
```
Comment (dalbertcloudsmith) [inline examples/alpine-cli/build.sh:27]: Should this be recursive? `-R`

**Review by dalbertcloudsmith — APPROVED** [review]
Added a couple questions. I didn't check the execution of each of the examples, but it looks sane to me.

File: examples/alpine-cli/build.sh (hunk 4 above)
Comment (caitriona-cloudsmith) [reply examples/alpine-cli/build.sh:27]: Happy to be mistaken but I don't think there is a specific recursive flag for `docker cp`? According to [this](https://docs.docker.com/engine/reference/commandline/container_cp/) the `-a` flag performs the copy recursively.

**Pull request merged**